site/en/docs/privacy-sandbox/user-agent/snippets/index.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'User-Agent Snippets'
+subhead: >
+  Code snippets to transform the current Chrome user-agent to the reduced format.
+description: >
+  Test your own sites and services against the reduced Chrome user-agent format by using a regular expression to transform the current Chrome user-agent to the reduced format.
+date: 2021-11-26
+authors:
+  - rowan_m
+---
+
+A collection of code snippets to let you transform the current Chrome user-agent
+string to the new reduced format. Please [submit versions for your preferred
+platforms and
+languages]https://github.com/GoogleChrome/developer.chrome.com//blob/main/site/en/docs/privacy-sandbox/user-agent/snippets/index.md)
+or let us know what would be useful to add!
```

Comment:
```suggestion
A collection of code snippets to let you transform the current Chrome user-agent
string to the new reduced format. [Submit versions for your preferred
platforms and
languages](https://github.com/GoogleChrome/developer.chrome.com//blob/main/site/en/docs/privacy-sandbox/user-agent/snippets/index.md)
or let us know what would be useful to add!
```

Reply:
No pleading with the readers, your writing is already polite enough :)

Reply:
Updating.

Reply:
We have a linter warning that catches it, but I get the warning blindness  https://github.com/GoogleChrome/developer.chrome.com/pull/1786#issuecomment-979964014